transitions/registration.js

```diff
@@ -265,6 +265,7 @@ module.exports = {
             if (err) {
                 callback(err);
             } else if (registrations.length) { // id collision, retry
+                logger.warn('Registration ID ' + id + ' is not unique, retrying...');
```

Comment:
For example there was some work towards monitoring the sentinel log to notify admins when errors are logged...